[IMP] accounting: fiscal year (#2850)
odoo/documentation: content/applications/finance/accounting/taxation/fiscal_year/close_fiscal_year.rst

Do a year end in Odoo (close a fiscal year)
===========================================

Before going ahead with closing a fiscal year, there are a few steps one should typically take to

> ensure that your accounting is correct, up to date, and accurate:

Reviewer: up-to-date

> - Make sure you have fully reconciled your **bank account(s)** up to year end and confirm that your

Reviewer: year-end

Do a year end in Odoo (close a fiscal year)
===========================================

> Before going ahead with closing a fiscal year, there are a few steps one should typically take to
> ensure that your accounting is correct, up-to-date, and accurate:

Reviewer: ```suggestion
Before going ahead with closing a fiscal year,  a few steps should be taken to
ensure that your accounting is correct, up-to-date, and accurate:
```

-  Make sure you have fully reconciled your **bank account(s)** up to year-end and confirm that your
   ending book balances match your bank statement balances.

-  Verify that all :guilabel:`Customer Invoice` have been entered and approved and that there are no
   more draft invoices.

-  Confirm that you have entered and agreed on all :guilabel:`Vendor bills`.

-  Validate all :guilabel:`Expenses`, ensuring their accuracy.

-  Corroborate that all :guilabel:`Received payments` have been encoded and recorded accurately.

-  Close :guilabel:`Suspense accounts`.

> -  Book all the :guilabel:`Depreciation` and :guilabel:`Deferred Revenues`.

Reviewer: general comments: add "see also" in this section or hyperlinks, to link to the related document pages?

- Run the optional :guilabel:`Payments Matching` feature under the :guilabel:`More` dropdown on the
  dashboard, validating any open :guilabel:`Vendor Bills` and :guilabel:`Customer Invoices` with
  their payments. This step is optional, however it may assist the year-end process if all
  outstanding payments and invoices are reconciled, and could lead finding errors or mistakes in the
  system.

- Your accountant/bookkeeper will likely verify your balance sheet items and book entries for:

 - Year-end manual adjustments.

 - **Work in Progress**.

 - **Depreciation Journal Entries**.

 - **Loans**.

 - **Tax adjustments**.

If your accountant/bookkeeper is going through end of the year auditing, they may want to have paper
copies of all balance sheet items (such as loans, bank accounts, prepayments, sales tax statements,
etc...) to compare these with your Odoo balances.

During this process, it is good practice to set the :guilabel:`Lock date for Non-Advisers` to the
last day of the preceding financial year, which is set under the accounting configuration. This way,
the accountant can be confident that nobody is changing the previous year transactions while

> auditing the books.

Reviewer: I would maybe add this as an important note

.. warning::
  On this screen, you also have a field :guilabel:`all users Lock date`. This lock date will
  prevent journal entries creation prior to the defined date for all users.
  This lock date is irreversible and cannot be removed.

.. note::
  The :guilabel:`all journal Lock date` prevents journal entry creation or modification prior the
  defined date except for advisor users.
  This lock date is reversible and can be removed.

Closing the fiscal year
=======================

There is no need to do a specific year end closing entry in order to close out your
:guilabel:`Profit and Loss statement`.
The reports are created in real-time, meaning that your :guilabel:`Profit and Loss statement`
corresponds directly with the year end-date you specify in Odoo. Therefore, any time you generate

> the :guilabel:`Income Statement`, the beginning date will correspond with the beginning of the

Reviewer: ```suggestion
the :guilabel:`Income Statement`, the beginning date corresponds with the beginning of the
```

 - Year-end manual adjustments.

 - **Work in Progress**.

 - **Depreciation Journal Entries**.

 - **Loans**.

 - **Tax adjustments**.

If your accountant/bookkeeper is going through end of the year auditing, they may want to have paper
copies of all balance sheet items (such as loans, bank accounts, prepayments, sales tax statements,
etc...) to compare these with your Odoo balances.

.. important::
  During this process, it is good practice to set the :guilabel:`Lock date for Non-Advisers` to the
  last day of the preceding financial year, which is set under the accounting configuration. This
  way, the accountant can be confident that nobody is changing the previous year transactions while
  auditing the books.

.. warning::
  On this screen, you also have a field :guilabel:`all users Lock date`. This lock date will
  prevent journal entries creation prior to the defined date for all users.
  This lock date is irreversible and cannot be removed.

.. note::
  The :guilabel:`all journal Lock date` prevents journal entry creation or modification prior the
  defined date except for advisor users.

>   This lock date is reversible and can be removed.

Reviewer 2: I can't find this field, and the description seems very similar to the **lock date for non-advisers** field, right? I removed it for now.

  - **Tax adjustments**.

If your accountant/bookkeeper is going through end of the year auditing, they may want to have paper
copies of all balance sheet items (such as loans, bank accounts, prepayments, sales tax statements,
etc...) to compare these with your Odoo balances.

> During this process, it is good practice to set the **Lock date for Non-Advisers** to the last day

Reviewer: :guilabel:?

Reviewer: add the path to set the lock date?

of the preceding financial year, which is set under the accounting configuration. This way, the
accountant can be confident that nobody is changing the previous year transactions
while auditing the books.

.. image:: media/close_fiscal_year01.png
   :align: center

.. warning::

>    On this screen, you also have a field :guilabel: `all users Lock date`. This lock date will

Reviewer: guilabel didn't work

   prevent journal entries creation prior to the defined date for all users.
   This lock date is irreversible and can’t be removed.

.. note::

>    The all journal entries lock date prevents journal entry creation or modification prior the

Reviewer: ```suggestion
   The "all journal entries" lock date prevents journal entry creation or modification prior the
```

   defined date except for advisor users.
   This lock date is reversible and can be removed.

Closing the fiscal year
=======================

> There is no need to do a specific year end closing entry in order to close out your Profit and Loss

Reviewer: ```suggestion
There is no need to do a specific year-end closing entry to close out your Profit and Loss
```

statement.
The reports are created in real-time, meaning that your Profit and Loss statement corresponds
directly with the year end-date you specify in Odoo. Therefore, any time you generate the

> **Income Statement**, the beginning date will correspond with the beginning of the **Fiscal Year**

Reviewer: ```suggestion
**Income Statement**, the beginning date corresponds with the beginning of the **Fiscal Year**
```

and the account balances will all be 0.

Once the accountant/bookkeeper has created the journal entry to allocate the

> **Current Year Earnings**, you should set the **Lock Date** to the last day of the fiscal year.

Reviewer: guilabel?

Making sure that before doing so, you confirm whether or not the current year earnings in the

> **Balance Sheet** is correctly reporting a 0 balance.

Reviewer: guilabel?
